result/uds.yaml

```diff
@@ -0,0 +1,101 @@
+components:
+  schemas:
+    User.Defined.Metrics.Request:
+      description: |-
+        The user defined metrics result request to the traffic generator
+      type: object
+      properties:
+        choice:
+          description: |-
+            The type
+          type: string
+          x-field-uid: 1
+          x-enum:
+            port_names:
+              x-field-uid: 1
+            flow_names:
+              x-field-uid: 2
+        port_names:
+          description: |-
+            Names of ports for which to return results.
+            An empty list will return all port row results.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - '/components/schemas/Port/properties/name'
+          x-field-uid: 2
+        flow_names:
+          description: |-
+            Names of flows for which to return results.
+            An empty list will return all flow row results.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - '/components/schemas/Flow/properties/name'
+          x-field-uid: 3
+        column_names:
```

Comment:
I don't quite see why we need column_names here. Wouldn't we always return a column for each stat that we requested? And user can reduce by not requesting specific UDS values?

Reply:
Introduced it so that it is at par with all the other stat requests , as all of them have column names , but it makes sense to not have it for UDS as the choice is right there with the users.
Removed column names